recipes/ada/all/conanfile.py

```diff
@@ -0,0 +1,101 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import get, copy, rmdir
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+import os
+
+required_conan_version = ">=1.53.0"
+
+class AdaConan(ConanFile):
+    name = "ada"
```

Comment:
the name is quite generic, as my first impression was that [ada](https://en.wikipedia.org/wiki/Ada_(programming_language)) is a programming language.